dbt/adapters/athena/impl.py

```diff
@@ -673,8 +674,15 @@ def _is_current_column(col: dict) -> bool:
             return False
         return True
 
+    @staticmethod
+    def _is_iceberg_table(table: dict) -> bool:
+        """Check if table type is iceberg."""
+        if table.get("Parameters", {}).get("table_type") == "ICEBERG":
+            return True
+        return False
```

Comment:
this is not needed, as you can leverage `get_table_type` https://github.com/dbt-athena/dbt-athena/blob/main/dbt/adapters/athena/impl.py#L486

Reply:
I agree with you, added TableType Enum